src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -513,8 +513,13 @@ void Unit::UpdateSplineMovement(uint32 t_diff)
     }
 
     if (arrived)
+    {
         DisableSpline();
 
+        if (movespline->HasAnimation())
+            SetAnimationTier(movespline->GetAnimation());
```

Comment:
`GetAnimationTier` perhaps?

Reply:
I think that's to follow the packet name and movespline setter helper
`movementSpline.Animation->AnimTier = splineFlags.getAnimationId();`